e2e/playwright.config.ts

```diff
@@ -0,0 +1,77 @@
+import { defineConfig } from '@playwright/test';
+import { AzureReporterOptions } from '@alex_neo/playwright-azure-reporter/dist/playwright-azure-reporter';
+import path from 'path';
+const envPath = path.resolve(__dirname, '.env');
+import dotenv from 'dotenv';
+dotenv.config({ path: envPath });
```

Comment:
We could also move the E2E-specific values into the `services/.env`-file, and just load that one here. (without the duplication of course..)

Reply:
Done!